Utilities/Rename-ChildItemsToAsciiRecursively/Rename-ChildItemsToAsciiRecursively.psm1

```diff
@@ -19,7 +19,7 @@ function Rename-ChildItemsToAsciiRecursively
 
             if ($item.Name -ne $asciiName)
             {
-                Write-Host ("Renaming `"$($item.FullName)`" to `"$asciiName`".")
+                Write-Verbose "Renaming `"$($item.FullName)`" to `"$asciiName`"."
```

Comment:
Let's add a non-verbose message here about the number of items that were renamed.

Reply:
I've rewritten the method to pipe out a new PSObject with the renames. (Also no need to manually recurse.) I think more fitting than a string message.
![image](https://user-images.githubusercontent.com/4007293/179977624-26f1f540-7dd2-4a94-9750-2008ad5e03f4.png)

Reply:
OK, thanks!